libbpf-cargo/src/gen/btf.rs

```diff
@@ -805,30 +807,22 @@ impl<'s> GenBtf<'s> {
             }
         }
 
-        writeln!(
-            def,
-            r#"#[derive(Debug, Copy, Clone, Default, PartialEq, Eq)]"#
-        )?;
-        writeln!(def, r#"#[repr({signed}{repr_size})]"#)?;
-        writeln!(
-            def,
-            r#"pub enum {name} {{"#,
-            name = self.anon_types.type_name_or_anon(&t),
-        )?;
+        writeln!(def, r#"#[derive(Debug, Copy, Clone, Default)]"#)?;
+        writeln!(def, r#"pub struct {enum_name}({signed}{repr_size});"#,)?;
```

Comment:
Let's tag this type as transparent, please:
```suggestion
        writeln!(def, r#"#[derive(Debug, Copy, Clone, Default)]"#)?;
        writeln!(def, r#"#[repr(transparent)]"#)?;
        writeln!(def, r#"pub struct {enum_name}({signed}{repr_size});"#)?;
```
(or so)